### PR TITLE
APPCAFE-04: Adding a SwiftUI Detail View

### PR DESCRIPTION
**Pull request by 1699kratos**
## ISSUE:
* https://trello.com/c/oCEByMWd

## FIX:
* Added a function in MenuViewController to present swiftui view 
* Use of environment variables to show info in swiftui view
* Shared a function to dismiss the swiftui view
* Made a swiftui view for the image and use it in the main swiftui view

## EVIDENCES:
image

### CODE REVIEW

File: CafePrices/CafePrices/Model/Item.swift [hunk 1]
```diff
@@ -6,11 +6,8 @@
 //
 
 import Foundation
-class Item {
-    var itemName: String
-    var itemImage: String
-    init(itemName: String, itemImage: String) {
-        self.itemName = itemName
-        self.itemImage = itemImage
-    }
```
Comment (hachiosaki) [inline CafePrices/CafePrices/Model/Item.swift:15]: why you did this?

**Review by hachiosaki — CHANGES_REQUESTED** [review]
Captura de Pantalla 2021-11-08 a la(s) 18 10 14

*could you remove the "<" before the Back?

*why does appear "Detalles..." instead of "Datalles"

*add a sign of pesos ($) after the price

File: CafePrices/CafePrices/Model/Item.swift (hunk 1 above)
Comment (1699kratos) [reply CafePrices/CafePrices/Model/Item.swift:15]: Sorry for the init delete
And for the view, i´ll do the changes, thanks

Comment (1699kratos) [reply CafePrices/CafePrices/Model/Item.swift:15]: Also "Detalles..." Was because i´d like it haha

**Pull request merged**